Improved freeCodeCamp

Ta1al opened the pull request:
- Added proper presence states using the document title instead of generic strings.
- Removed presence update for `/login` path.

- Updated contributors and bumped version from `1.1.3` to `1.1.5`

Ta1al committed:
Improved freeCodeCamp

Ta1al committed:
Updated contributors

Ta1al committed:
Updated version

Ta1al committed:
Removed unused param

EncryptedDev reviewed (commented):
Upload proof that it is working

Ta1al committed:
Shortened code

Ta1al commented:
I have just thought of something else and I can make this presence even better

Ta1al committed:
Improved freeCodeCamp

- Added a button
- Added presence state for `settings` and `profile` page
- Improved presence for `learn` page

Ta1al committed:
Added /donate page

Ta1al commented:
- Added better support for `/learn`
- Added support for `/settings`, `/donate` and profile page
- Added `View Page` button `(URL = "https://freecodecamp.org" + document.location.pathname)`

**Proof of working:**

`/`:
![mainPage](https://user-images.githubusercontent.com/43641182/137621449-b1e11263-bb9d-4313-bf19-cd51253c8b5a.png)
`/learn`:
![learnMainPage](https://user-images.githubusercontent.com/43641182/137621476-8205822b-f864-4b7a-b13c-32dd7be87b65.png)
`/learn/[course]`:
![learnCourse](https://user-images.githubusercontent.com/43641182/137621494-2331c211-b90d-4d29-a090-ccbaa8c8b7d5.png)
`/learn/[course]/[lesson]`:
![learnLesson](https://user-images.githubusercontent.com/43641182/137621499-5c5a0744-97fa-4c2f-8a11-312deba552a1.png)
`/news`:
![newsMain](https://user-images.githubusercontent.com/43641182/137621504-3df17cfb-3405-4666-a567-0a91a32374d3.png)
`/news/[article]`:
![newsArticle](https://user-images.githubusercontent.com/43641182/137621507-0e58e43a-85ba-468a-b977-e13ad2f11ca7.png)
`/settings`:
![settings](https://user-images.githubusercontent.com/43641182/137621509-496e39b1-f2bb-4373-881d-07760e394f85.png)
`/donate`:
![donate](https://user-images.githubusercontent.com/43641182/137621514-f6f2cb2c-56ea-4832-84b2-c48fd0cbef8b.png)
`/[profile]`:
![profile](https://user-images.githubusercontent.com/43641182/137621524-659a01a2-d7fd-4e22-ba4a-38220975a62a.png)

Ta1al committed:
Updated freeCodeCamp
